roles/beaker/data/distros/tasks/main.yml

```diff
@@ -11,13 +11,14 @@
     - http://mirror.centos.org/centos/6/os/x86_64/
     - http://mirror.centos.org/centos/6/os/i386/
     - http://mirror.centos.org/centos/7/os/x86_64/
+    - http://mirror.centos.org/centos/8/BaseOS/x86_64/os/
```

Comment:
This is not going to work as we need. CentOS 8 compose metadata are incomplete (and CentOS is ignoring me)

Reply:
I see. I removed it.